src/app/shared/components/filter/filter-navigation/filter-navigation.component.ts

```diff
@@ -14,20 +16,27 @@ import { FilterNavigation } from 'ish-core/models/filter-navigation/filter-navig
 export class FilterNavigationComponent implements OnInit {
   @Input() fragmentOnRouting: string;
   @Input() orientation: 'sidebar' | 'horizontal' = 'sidebar';
+  @Input() showCategoryFilter = true;
 
   filter$: Observable<FilterNavigation>;
 
   constructor(private shoppingFacade: ShoppingFacade, private router: Router, private activatedRoute: ActivatedRoute) {}
 
   ngOnInit() {
-    this.filter$ = this.shoppingFacade.currentFilter$;
+    this.filter$ = this.shoppingFacade.currentFilter$.pipe(
+      whenTruthy(),
+      map(x =>
+        this.showCategoryFilter ? x : { ...x, filter: x.filter.filter(f => f.id !== 'CategoryUUIDLevelMulti') }
+      )
```

Comment:
This logic is only used here (as is the facade property), so I guess it could be moved into the facade.